shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/CompiledGenerator.java

```diff
@@ -147,66 +142,56 @@ public void errorHtml(SectionRenderer renderer) {
 	public void run(ActionConsumer consumer, InputProvider input) {
 		// Load all the input data in an attempt to cache it before any olives try to
 		// use it. This avoids making the first olive seem really slow.
-		Stream.<LimitedRunnable>concat(InputFormatDefinition.formats()//
-				.map(format -> new LimitedRunnable() {
-
-					@Override
-					public String name() {
-						return format.name();
-					}
-
-					@Override
-					public void run() {
-						try {
-							INPUT_RECORDS.labels(format.name()).set(format.input(format.itemClass()).count());
+		// TODO: collect only the formats that olives are currently using
```

Comment:
Is there a ticket for this `TODO` (ie. can it be removed from the code)?

Reply:
There's a ticket, but, I would leave it in...